proto/hypertrace/agent/config/v1/config.proto

```diff
@@ -134,6 +137,9 @@ enum MetricReporterType {
     // Prometheus exposition format.
     // see https://github.com/prometheus/docs/blob/main/content/docs/instrumenting/exposition_formats.md
     METRIC_REPORTER_TYPE_PROMETHEUS = 2;
+
+    // Disable metric reporting
+    METRIC_REPORTER_TYPE_NONE = 3;
```

Comment:
while we're here, should we also add a logging exporter?

Reply:
Curious about logger exporting for metrics. Is that a thing?

Reply:
yep, at least for the collector.

https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/loggingexporter/README.md

Reply:
Also for java: https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md#logging-exporter

Reply:
yes, done.